src/lib/zf/zf_stats.c

```diff
@@ -0,0 +1,69 @@
+/* SPDX-License-Identifier: MIT */
+/* SPDX-FileCopyrightText: (c) Advanced Micro Devices, Inc. */
+#include <zf_internal/zf_stack.h>
+
+/* ensure ef_vi/zf structures are the same size in case
```

Comment:
When it's used with ef_vi_transmit_alt_overhead that is because the user application will be making an ef_vi call. That's not the case here and I wanted to avoid having a user application pull in vi.h just to get two structure definitions.